test/e2e/network/endpointslice.go

```diff
@@ -126,7 +126,7 @@ var _ = SIGDescribe("EndpointSlice", func() {
 		}
 
 		err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
-		framework.ExpectNoError(err)
+		framework.ExpectNoError(err, "error deleting Service")
 
 		// Expect Endpoints resource to be deleted when Service is.
 		if err := wait.PollImmediate(2*time.Second, 12*time.Second, func() (bool, error) {
```

Comment:
same question about why we're picking such a specific duration

Reply:
Updated, thanks!